SDKCLI/Sources/SDKCLI/Models/Version.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+struct Version {
+    var major: Int
+    var minor: Int
+    var patch: Int
+    
+    init(_ versionString: String) throws {
+        let versions = versionString
+            .trimmingCharacters(in: .whitespacesAndNewlines)
+            .components(separatedBy: ".")
+        guard
+            versions.count == 3,
+            let major = Int(versions[0]),
+            let minor = Int(versions[1]),
+            let patch = Int(versions[2])
+        else {
+            throw Error("Failed to parse version from string \(versionString)")
+        }
+        
+        self.major = major
+        self.minor = minor
+        self.patch = patch
+    }
+    
+    var versionString: String { "\(major).\(minor).\(patch)" }
+    
+    mutating func incrementMajor() {
```

Comment:
I expect this package in same category as SDK, we must add docs for public APIs. They could be useful for other than just us.